samples/web_only.yml

```diff
@@ -28,8 +33,8 @@ env:
   DISCOURSE_DB_SOCKET: ''
   #DISCOURSE_DB_USERNAME: discourse
   DISCOURSE_DB_PASSWORD: SOME_SECRET
-  DISCOURSE_DB_HOST: DB_IP_ADDRESS
-  DISCOURSE_REDIS_HOST: REDIS_IP_ADDRESS
+  DISCOURSE_DB_HOST: DB_CONTAINER_ALIAS
+  DISCOURSE_REDIS_HOST: REDIS_CONTAINER_ALIAS
```

Comment:
hmm how is this going to work? who is setting DB_CONTAINER_ALIAS ? 

if only a container named "data" is being linked it would probably simply be "data" there.